ui/components-react/src/test/filter-builder/FilterBuilderRuleGroup.test.tsx

```diff
@@ -129,123 +95,4 @@ describe("PropertyFilterBuilderRuleGroupRenderer", () => {
 
     getByDisplayValue("Prop");
   });
-
-  it("renders child rule group", () => {
-    const property: PropertyDescription = {
-      displayLabel: "Prop",
-      name: "prop",
-      typename: "int",
-    };
-    const { getByText } = renderWithContext(
-      <PropertyFilterBuilderRuleGroupRenderer
-        {...defaultProps}
-        group={{
-          id: "id",
-          operator: PropertyFilterRuleGroupOperator.And,
-          items: [
-            {
-              id: "childId",
-              groupId: "id",
-              operator: PropertyFilterRuleGroupOperator.Or,
-              items: [
-                {
-                  id: "grandChildId1",
-                  groupId: "childId",
-                  property,
-                },
-                {
-                  id: "grandChildId2",
-                  groupId: "childId",
-                  property,
-                },
-              ],
-            },
-          ],
-        }}
-      />
-    );
-
-    getByText(
-      TestUtils.i18n.getLocalizedString("Components:filterBuilder.operators.or")
-    );
-  });
-
-  it("dispatches add rule event when button is clicked", () => {
-    const actions = new PropertyFilterBuilderActions(sinon.spy());
-    const { getByTestId } = renderWithContext(
-      <PropertyFilterBuilderRuleGroupRenderer {...defaultProps} />,
-      { actions }
-    );
-    const addItemSpy = sinon.stub(actions, "addItem");
-
-    const addRuleButton = getByTestId("rule-group-add-rule");
-    addRuleButton.click();
-
-    expect(addItemSpy).to.be.calledOnceWith(defaultProps.path, "RULE");
-  });
-
-  it("dispatches add rule group event when button is clicked", () => {
-    const actions = new PropertyFilterBuilderActions(sinon.spy());
-    const { getByTestId } = renderWithContext(
-      <PropertyFilterBuilderRuleGroupRenderer {...defaultProps} />,
-      { actions }
-    );
-    const addItemSpy = sinon.stub(actions, "addItem");
-
-    const addRuleGroupButton = getByTestId("rule-group-add-rule-group");
-    addRuleGroupButton.click();
-
-    expect(addItemSpy).to.be.calledOnceWith(defaultProps.path, "RULE_GROUP");
-  });
-
-  it("dispatches remove item event when button is clicked", () => {
-    const actions = new PropertyFilterBuilderActions(sinon.spy());
-    const { getByTestId } = renderWithContext(
-      <PropertyFilterBuilderRuleGroupRenderer
-        {...defaultProps}
-        group={{
-          id: "id",
-          groupId: "parentGroupId",
-          operator: PropertyFilterRuleGroupOperator.And,
-          items: [],
-        }}
-      />,
-      { actions }
-    );
-    const removeItemSpy = sinon.stub(actions, "removeItem");
-
-    const removeButton = getByTestId("rule-group-remove");
-    removeButton.click();
-
-    expect(removeItemSpy).to.be.calledOnceWith(defaultProps.path);
-  });
-
-  it("dispatches operator change event when operator is selected", async () => {
```

Comment:
I think this test should not be removed. We still need to check if operator changes for root group that is render.